starter-code/main.js

```diff
@@ -0,0 +1,250 @@
+const MongoDB = require('mongodb');
+const mongoClient = MongoDB.MongoClient;
+const clear = require('clear');
+const readline = require('readline');
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+});
+
+const url = `mongodb://localhost:27017/crunchdatabase`;
+
+mongoClient.connect(url, (error, db) => {
+  if (error) {
+    console.log('Error trying to connect to the Database');
+    console.log(error);
+  } else {
+    console.log('Connection established correctly!! 😬');
+
+    function mainMenu() {
+      clear();
+      printMenu();
+      rl.question('Type an option: ', (option) => {
+        switch (option) {
+          case "1":
+          db.collection('companies').find({}, {
+              name: 1,
+              _id: 0
+            }).toArray((error, result) => {
+              if (error) {
+                console.log(error);
+                rl.question(`\nType enter to continue: `, (answer) => {
+                  mainMenu();
+                });
+              } else {
+                console.log(result);
+                rl.question(`\nType enter to continue: `, (answer) => {
+                  mainMenu();
+                });
+              }
+            });
+            break;
+          case "2":
+          db.collection('companies').count((error, result) => {
+              if (error) {
+                console.log(error);
+                rl.question(`\nType enter to continue: `, (answer) => {
+                  mainMenu();
+                });
+              } else {
+                console.log(result);
+                rl.question(`\nType enter to continue: `, (answer) => {
+                  mainMenu();
+                });
+              }
+            });
+            break;
+          case "3":
+          db.collection('companies').find({founded_year:2004}, {
```

Comment:
Por si quieres ahorrar código, el filtro por el que se busca se puede poner en el count y omitir el find:
`db.collection('companies').count({"founded_year": 2004}, (error, result)=>{
						if (error) { 
							console.log(error);
							rl.question(`\nType enter to continue: `, (answer) => {mainMenu()});
						} else {
							console.log(`Number of companies founded in 2004: ${result}`);
							rl.question(`\nType enter to continue: `, (answer) => {mainMenu()});	
						}
					});`